tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/mapper/ConditionsOfJoiningMapperTest.java

```diff
@@ -0,0 +1,46 @@
+package com.transformuk.hee.tis.tcs.service.mapper;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertNotNull;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.transformuk.hee.tis.tcs.api.dto.ConditionsOfJoiningDto;
+import com.transformuk.hee.tis.tcs.api.enumeration.GoldGuideVersion;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.UUID;
+import org.junit.Test;
+
+public class ConditionsOfJoiningMapperTest {
+
+  @Test
+  public void shouldIgnoreUnknownPropertiesWhenDeserializing() throws JsonProcessingException {
+
+    UUID uuid = UUID.randomUUID();
+    Instant signedAt = Instant.now();
+    GoldGuideVersion goldGuideVersion = GoldGuideVersion.GG9;
+
+    String jsonAsString =
+        "{\"programmeMembershipUuid\":\"" + uuid.toString() + "\"," +
```

Comment:
I was about to say how a text block would be more readable, then I remembered where we were 🙊